src/server/pdc_server_region/pdc_server_region_cache.c

```diff
@@ -6,10 +6,10 @@
 
 #ifdef PDC_SERVER_CACHE
 
-#ifdef PDC_SERVER_CACHE_MAX_SIZE
-#define MAX_CACHE_SIZE PDC_SERVER_CACHE_MAX_GB * 1024 * 1024 * 1024
+#ifdef PDC_SERVER_CACHE_MAX_GB
+#define MAX_CACHE_SIZE_GB PDC_SERVER_CACHE_MAX_GB
```

Comment:
We need to make sure the docs are updated correctly with the env name and unit people should use.